### DISCUSSION TITLE
zkSYNC Showroom

### DISCUSSION THREAD

**Discussion by emresen30** [zkSync-Community-Hub/zksync-developers#327] [Show and Tell]
### Project Name

zkShow

### Technologies Used

zkSYNC Era Mainnet

### Describe your project. What does it do? Why did you create it? Any interesting challenges you faced?

Many models may take part in the event. These models can be developed with tasks to be performed accordingly. Many features unique to the zkSYNC network can be added to the character.

### What specific feedback are you looking for?

character edits

### How can the community help or contribute to your project?

Many improvements should be made in line with the ideas of the community.

**Comment by hapizddcr** [comment]
👀

**Comment by MexicanAce** [comment]
Hi @emresen30 , is this a submission that exists already and is available for review? Typically `Show and Tell` submissions already have an application deployed and are a place for feedback

**Comment by jmwhangjin** [comment]
Great
